spec/spec.md

```diff
@@ -682,12 +682,22 @@ Examples are listed inline in above sections as well as in complete form within
 
 - Microsoft
 - Workday
-- Mattr
 - Ping Identity
+- Gen Digital
+- Spruce Systems Inc.
+- Auth0Lab
 
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://app.vp.interop.spruceid.xyz/
+- https://verifycredential.pingone.com/verifier.html
```

Comment:
For Ping, please change this to https://vc.ping-eng.com/verifier.html

Reply:
```suggestion
- https://vc.ping-eng.com/verifier.html
```